packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -56,9 +49,10 @@ export default {
 			}
 		},
 		formError() {
-			if (this.currentValue > this.max && this.max !== null) return true
-			if (this.currentValue < this.min && this.min !== null) return true
-			if (this.currentValue === '' || isNaN(this.currentValue)) return true
+			if (this.max !== null && this.currentValue > this.max) return true
+			if (this.min !== null && this.currentValue < this.min) return true
+			if (typeof this.currentValue === 'string' || isNaN(this.currentValue))
```

Comment:
I don't follow - 
max and min are both default null, so i have to check for this in case one of them is defined as 0., and because i want to regard them as numbers.  

the typeof current value string is indeed redundant - so i'll remove it. 

type is number for value - in the vue props will be a false positive if NaN is actually passed from without, or the user types in a value that will be parsed as NaN. So I have to check this - no?

I could actually move most of the logic to validators in the vue props - but we need this as a computed property to show the red border on the input from the html.  

min and max are both default null.